test/content/support.js

```diff
@@ -241,4 +241,17 @@ function resetDB() {
 	}).then(function() {
 		return Zotero.Schema.schemaUpdatePromise;
 	});
-}
\ No newline at end of file
+}
+
+
+/**
+ * Imports an attachment from a test file.
+ * @param {string} filename - The filename to import (in data directory)
+ * @returns {Promise} as returned by Zotero.Attachments.importFromFile
+ */
+function importFromTestFile(filename) {
```

Comment:
(I think this is meant to mirror importFromFile, so it's clear if you're familiar with the Zotero.Attachments API, but in other contexts importFileAttachment is probably clearer.)